catchqr.iml

```diff
@@ -8,10 +8,11 @@
       <excludeFolder url="file://$MODULE_DIR$/.dart_tool" />
       <excludeFolder url="file://$MODULE_DIR$/.idea" />
       <excludeFolder url="file://$MODULE_DIR$/build" />
+      <excludeFolder url="file://$MODULE_DIR$/.pub" />
```

Comment:
Good point here actually. Shouldn't we exclude same files in toPWr @simon-the-shark?

Reply:
sure, we can